naming/README.md

```diff
@@ -0,0 +1,132 @@
+# IPNS - Inter-Planetary Naming System
+
+Authors:
+
+  - Vasco Santos ([@vasco-santos](https://github.com/vasco-santos))
+  
+Reviewers:
+
+  - Steven Allen ([@Stebalien](https://github.com/Stebalien))
+
+-----
+  
+# Abstract
+
+IPFS is powered by content-addressed data, which by nature is immutable: changing an object would change its hash, and consequently its address, making it a different object altogether. However, there are several use cases where we benefit from having mutable data. This is where IPNS gets into the equation.
+
+All things considered, the IPFS naming layer is responsible for the creation of:
+
+  - mutable pointers to objects
+  - human-readable names
+
+# Status of this spec
+
+![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+# Organization of this document
+
+  - [Introduction](#introduction)
+  - [IPNS Record](#ipns-record)
+  - [Protocol](#protocol)
+  - [Overview](#overview)
+  - [API Spec](#api-spec)
+  - [Integration with IPFS](#integration-with-ipfs)
+
+# Introduction
+
+Each time a file added to IPFS needs to be modified, its content addressable idendifier will be automatically modified. As a consequence, the address previously used for getting that file needs to be updated by who is using it. As this is not pratical, IPNS was conceptualized to solve the problem.
```

Comment:
```suggestion
Each time a file is modified, its content address changes. As a consequence, the address previously used for getting that file needs to be updated by who is using it. As this is not pratical, IPNS was created to solve the problem.
```

Wording nits:

* IPFS doesn't react to need, unfortunately.
* IPNS is more than a concept (it exists).


"needs to be updated by who is using it" is still awkward but I'm not sure of a better wording off the top of my head.